subtests/docker_cli/cp/cp.py

```diff
@@ -207,3 +208,72 @@ def verify_files_number(self, copied_number, expected_number):
         self.loginfo("Success, copied %d files from container, "
                      "expected number from configuration %d"
                      % (copied_number, expected_number))
+
+
+class volume_mount(CpBase):
+    """
+    Regression between docker-1.10.3 and 1.12.
+
+    See: https://github.com/docker/docker/issues/27773
+    """
+
+    def initialize(self):
+        super(volume_mount, self).initialize()
+
+        vol = 'v_' + utils.generate_random_string(8)
+        subargs = ['create', '--name', vol]
+        mustpass(DockerCmd(self, 'volume', subargs).execute())
+        self.sub_stuff['volume_name'] = vol
+
+        # First container: bind-mounts new volume, then unpacks two
+        # tarballs into it. I don't know why it has to be two tarballs
+        # and not one, but the tar thing (vs plain copy) has to do
+        # with pivot_root. Whatever that is.
+        c1 = 'c1_' + utils.generate_random_string(8)
```

Comment:
``dockertest.containers.DockerContainers.get_unique_name()`` makes sure name's characters conform and that it doesn't already "randomly" exist.